### PR TITLE
Release zenko 2.6.65

### PR DESCRIPTION
**Pull request by francoisferrand**
- Bump pensieve-api 1.6.0
- Bump vault 8.8.8
- Bump backbeat 8.6.48
- Bump zenko-operator 1.6.0
- Bump sorbet 1.1.11
- Add continue-on-error to kafka ci dump
- Move external tools to deps.yaml
- Add drctl
- Add haproxy
- Use sharded mongo everywhere
- Remove patching of PRA zenkoversion
- Remove mongodb (non-sharded) and kubedb from ISO
- Add missing debug steps in PRA job
- Add DrCTL to devcontainer ctst
- Precreate DR kafka volume
- Release Zenko 2.6.64

Issue: ZENKO-4856

### CODE REVIEW

File: monitoring/pra/dashboard.py [hunk 1]
```diff
@@ -0,0 +1,311 @@
...
+        legendDisplayMode="list",
+        legendCalcs=["max", "mean"],
+        unit=UNITS.SECONDS,
+        targets=[Target(
+            expr="sum(" + metric + ")",
+            legendFormat=" ",
+        )],
+    )
+    for name, metric in [
+        ["Source", Metrics.KAFKA_CONNECT_SOURCE_TASK_RECORDS_LAG(Metrics.DR_SOURCE, job="${kafka_connect_src_job}")],
```
Comment (welansari) [inline monitoring/pra/dashboard.py:168]: Not sure this metric makes sense for the source connector as it doesn't consume from Kafka

**Review by welansari — APPROVED** [review]

File: .devcontainer/setup.sh [hunk 2]
```diff
@@ -1,22 +1,22 @@
...
 export GIT_ACCESS_TOKEN=${GITHUB_TOKEN}
 export E2E_IMAGE_TAG=latest
 
-array_length=`yq ".runs.steps | length - 1" .github/actions/deploy/action.yaml`
+array_length=$(yq ".runs.steps | length - 1" .github/actions/deploy/action.yaml)
 for i in $(seq 0 $array_length); do
-    step=`yq ".runs.steps[$i]" .github/actions/deploy/action.yaml`
-    working_dir=`yq ".runs.steps[$i].working-directory" .github/actions/deploy/action.yaml`
-    run_command=`yq ".runs.steps[$i].run" .github/actions/deploy/action.yaml`
+    #step=$(yq ".runs.steps[$i]" .github/actions/deploy/action.yaml)
```
Comment (benzekrimaha) [inline .devcontainer/setup.sh:11]: was this commented on purpose ?

Comment (francoisferrand) [reply .devcontainer/setup.sh:11]: yes, I will followup and fix this with TLS...

**Review by benzekrimaha — APPROVED** [review]
LGTM

**Review by KillianG — APPROVED** [review]

**Pull request merged**